lib/netext/grpcext/conn_test.go

```diff
@@ -259,9 +259,17 @@ func (m *getServiceFileDescriptorMock) Recv() (*reflectpb.ServerReflectionRespon
 }
 
 func methodFromProto(method string) protoreflect.MethodDescriptor {
+	path := "any-path"
 	parser := protoparse.Parser{
 		InferImportPaths: false,
 		Accessor: protoparse.FileAccessor(func(filename string) (io.ReadCloser, error) {
+			// a small hack to make sure we are parsing the right file
+			// otherwise the parser will try to parse "google/protobuf/descriptor.proto"
+			// with exactly the same name as the one we are trying to parse for testing
+			if filename != path {
+				return nil, nil
+			}
```

Comment:
Why is it trying to load a different file?

Reply:
It's coming from the library update https://github.com/bufbuild/protocompile/commit/7c5114ee37b44b274e401c07b53f3fbfadf81dbb